Filebeat exits with error when ES module is enabled on Mac OS X

- Version: `master`
- Operating System: Mac OS X
- Steps to Reproduce: 
   1. Make sure you are on Mac OS X.
   2. Enable the `elasticsearch` module in Filebeat:
      ```
      filebeat modules enable elasticsearch
      ```
   3. Start Filebeat with logs going to console (for easy verification):
      ```
      filebeat -e
      ```

- Observed behavior:
  Filebeat will exit with this error:
  ```
  2018-11-01T09:40:39.632-0700    INFO    crawler/crawler.go:72   Loading Inputs: 1
  2018-11-01T09:40:39.645-0700    INFO    log/input.go:138        Configured paths: [/usr/local/elasticsearch/*_access.log]
  2018-11-01T09:40:39.646-0700    INFO    log/input.go:138        Configured paths: [/usr/local/elasticsearch/*_deprecation.log]
  2018-11-01T09:40:39.646-0700    INFO    log/input.go:138        Configured paths: [/usr/local/elasticsearch/gc.log.[0-9]* /usr/local/elasticsearch/gc.log]
  2018-11-01T09:40:39.646-0700    ERROR   fileset/factory.go:105  Error creating input: No paths were defined for input accessing config
  2018-11-01T09:40:39.646-0700    INFO    crawler/crawler.go:139  Stopping Crawler
  2018-11-01T09:40:39.646-0700    INFO    crawler/crawler.go:149  Stopping 0 inputs
  2018-11-01T09:40:39.646-0700    INFO    crawler/crawler.go:165  Crawler stopped
  2018-11-01T09:40:39.646-0700    INFO    registrar/registrar.go:356      Stopping Registrar
  2018-11-01T09:40:39.646-0700    INFO    registrar/registrar.go:282      Ending Registrar
  2018-11-01T09:40:39.648-0700    INFO    [monitoring]    log/log.go:152  Total non-zero metrics  {"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":21,"time":{"ms":21}},"total":{"ticks":53,"time":{"ms":53},"value":53},"user":{"ticks":32,"time":{"ms":32}}},"info":{"ephemeral_id":"03f45ea2-99ac-4ef8-958f-0bcd6a9e44b3","uptime":{"ms":3032}},"memstats":{"gc_next":4194304,"memory_alloc":1826016,"memory_total":4557336,"rss":20090880}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"output":{"type":"elasticsearch"},"pipeline":{"clients":0,"events":{"active":0}}},"registrar":{"states":{"current":1},"writes":{"success":1,"total":1}},"system":{"cpu":{"cores":8},"load":{"1":2.248,"15":2.4346,"5":2.1929,"norm":{"1":0.281,"15":0.3043,"5":0.2741}}}}}}
  2018-11-01T09:40:39.648-0700    INFO    [monitoring]    log/log.go:153  Uptime: 3.033037303s
  2018-11-01T09:40:39.648-0700    INFO    [monitoring]    log/log.go:130  Stopping metrics logging.
  2018-11-01T09:40:39.648-0700    INFO    instance/beat.go:432    filebeat stopped.
  2018-11-01T09:40:39.648-0700    ERROR   instance/beat.go:816    Exiting: No paths were defined for input accessing config
  Exiting: No paths were defined for input accessing config
  ```

  This happens probably because on Mac OSX the `elasticsearch/server` fileset has no file paths defined: https://github.com/elastic/beats/blob/cfdea3db83edd7796868390c8b20ec0d3ce9d9c5/filebeat/module/elasticsearch/server/manifest.yml#L7

  (The same thing probably happens on Windows but I haven't tested it). 

- Expected behavior:
  Given that the user hasn't done any special configuration yet (apart from enabling the Elasticsearch module), I'd expect things to Just Work:tm: out of the box.

--- a/CHANGELOG.asciidoc
+++ b/CHANGELOG.asciidoc
@@ -77,6 +77,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Fix RFC3339 timezone and nanoseconds parsing with the syslog input. {pull}8346[8346]
 - Support multiline logs in logstash/log fileset of Filebeat. {pull}8562[8562]
 - Fix improperly set config for CRI Flag in Docker Input {pull}8899[8899]
+- Just enabling the `elasticsearch` fileset and starting Filebeat no longer causes an error. {pull}8891[8891]
 
 *Heartbeat*
 

--- a/filebeat/module/elasticsearch/server/manifest.yml
+++ b/filebeat/module/elasticsearch/server/manifest.yml
@@ -4,8 +4,10 @@ var:
   - name: paths
     default:
       - /var/log/elasticsearch/*.log
-    os.darwin: []
-    os.windows: []
+    os.darwin:
+      - /usr/local/elasticsearch/*.log
+    os.windows:
+      - c:/ProgramData/Elastic/Elasticsearch/logs/*.log
 
 ingest_pipeline: ingest/pipeline.json
 prospector: config/log.yml

